modules/clients/src/test/java/org/apache/ignite/internal/processors/rest/JettyRestProcessorAbstractSelfTest.java

```diff
@@ -1636,6 +1644,190 @@ public void testExe() throws Exception {
+            .addArguments(DEFAULT_CACHE_NAME, "{\"type\":\"" + VisorDataType.INT + "\",\"value\":\"1\"}"));
+
+        info("VisorCacheGetValueTask result for Integer key: " + ret);
+
+        JsonNode res = jsonTaskResult(ret);
+
+        String resStr = res.get("result").toString();
+
+        assertTrue(resStr.contains("\"id\":" + p.getId()));
+        assertTrue(resStr.contains("\"orgId\":" + p.getOrganizationId()));
+        assertTrue(resStr.contains("\"firstName\":\"" + p.getFirstName() + "\""));
+        assertTrue(resStr.contains("\"lastName\":\"" + p.getLastName() + "\""));
+        assertTrue(resStr.contains("\"salary\":" + p.getSalary()));
+
+        // Check UUID key.
+        UUID uuidKey = UUID.randomUUID();
+
+        jcache().put(uuidKey, 2);
+
+        ret = content(new VisorGatewayArgument(VisorCacheGetValueTask.class)
+            .setNode(locNode)
+            .setTaskArgument(VisorCacheGetValueTaskArg.class)
+            .addArguments(DEFAULT_CACHE_NAME, "{\"type\":\"" + VisorDataType.UUID + "\",\"value\":\"" + uuidKey.toString() + "\"}"));
+
+        info("VisorCacheGetValueTask result for UUID key: " + ret);
+
+        res = jsonTaskResult(ret);
+
+        resStr = res.get("result").toString();
+
+        assertTrue("2".equals(resStr));
+
+        // Check Timestamp key.
+        long timestamp = System.currentTimeMillis();
+
+        jcache().put(new Timestamp(timestamp), 3);
+
+        ret = content(new VisorGatewayArgument(VisorCacheGetValueTask.class)
+            .setNode(locNode)
+            .setTaskArgument(VisorCacheGetValueTaskArg.class)
+            .addArguments(DEFAULT_CACHE_NAME, "{\"type\":\"" + VisorDataType.TIMESTAMP + "\",\"value\":\"" + timestamp + "\"}"));
+
+        info("VisorCacheGetValueTask result for Timestamp key: " + ret);
+
+        res = jsonTaskResult(ret);
+
+        resStr = res.get("result").toString();
+
+        assertTrue("3".equals(resStr));
+
+        // Check Date key.
+        //jcache().put(new Date(timestamp), 4);
+
+        //ret = content(new VisorGatewayArgument(VisorCacheGetValueTask.class)
+        //    .setNode(locNode)
+        //    .setTaskArgument(VisorCacheGetValueTaskArg.class)
+        //    .addArguments(DEFAULT_CACHE_NAME, "{\"type\":\"" + VisorDataType.DATE_SQL + "\",\"value\":\"" + timestamp + "\"}"));
+        //
+        //info("VisorCacheGetValueTask result for Date key: " + ret);
+        //
+        //res = jsonTaskResult(ret);
+        //
+        //resStr = res.get("result").toString();
+        //
+        //assertTrue("4".equals(resStr));
+
+        // Check object key.
+        jcache().put(p, 5);
+
+        ret = content(new VisorGatewayArgument(VisorCacheGetValueTask.class)
+            .setNode(locNode)
+            .setTaskArgument(VisorCacheGetValueTaskArg.class)
+            .addArguments(DEFAULT_CACHE_NAME,
+                "{" +
+                    "\"type\":\"" + VisorDataType.BINARY + "\"," +
```

Comment:
Maybe `String.format()` would help to save some space and would be better to read

Reply:
Changed.